blog/_posts/2020-02-12-local-dev.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: local-dev
+date: 2020-02-11
+author: dan
+layout: blog
+title: "Non-Kubernetes Local Dev"
```

Comment:
Title suggestion:

"Tilt is a Better Foreman"

"Tilt: Supercharged Foreman"

I think we want an attention grabbing title.

Reply:
WDYT of "Tilt Beyond Kubernetes"?

Argh, I think that's too grandiose...

Reply:
But, like, the point is that we're expanding...

Reply:
IMO The title's job is to grab your attention to you click in to the article, not to describe the main point of the article. :P

I think "Tilt Beyond Kubernetes" is an improvement, FWIW.